rebar.config

```diff
@@ -7,7 +7,7 @@
 {deps, [
         {exml, ".*", {git, "git://github.com/esl/exml.git", {tag, "2.1.5"}}},
         {base16, ".*", {git, "git://github.com/goj/base16.git", "ec420aa"}},
-        {fusco, ".*", {git, "git://github.com/esl/fusco.git", {branch, "master"}}},
+        {fusco, ".*", {git, "git://github.com/esl/fusco.git", {branch, "0a428471"}}},
```

Comment:
Correct apologies - fixed locally now, it fetches the deps and I will force-push now.